executor.c

```diff
@@ -10,11 +10,13 @@
 
 int executor(char **args, char **env)
 {
-	int success = 0;
-	
-	success = execve(args[0], args, env);
-	if (success < 0)
-		return (0);
+	pid_t childp = fork();
+
+	if (childp != 0)
```

Comment:
this should be before `execev`, u also to handle both child and father, if u in child process, run execev, if u in father process, just wait for child to finish, and determine if the execution was successful in order to return the correct value, `execev` should only be called by child